Makefile

```
@@ -391,6 +391,11 @@ lint: golangci-lint
 
 lint-fix: golangci-lint-fix
 
+check-permissions: # Check if files are not executable expect for allowlisted files. \
```

Non-blocking nit:

```suggestion
check-permissions: ## Check if files are not executable expect for allowlisted files. \
```

If we want the doc in `make help`

---

good idea, thanks!